testdata/scripts/literals.txt

```diff
@@ -75,10 +75,13 @@ func main() {
 	testMap["map key"] = "new value"
 	println(testMap["map key"])
 	println("another literal")
-	println(skip1, skip2)
-	println(i, foo, bar)
+	println(skip2)
+	println(boolean, i, foo, bar)
 	typedTest()
 	constantTest()
+	intTest()
+	byteTest()
+	boolTest()
```

Comment:
I'm not sure that all these extra tests are really testing much. You're just testing that they work as expected, but that will be the case without literal obfuscation too.

You should be checking that you can't find the original value in the binary. For example, if you have a large integer like 12345678, figure out how that would look in the binary on amd64 (e.g. little endian), and check that it exists in a normal build, but doesn't with `garble -literals`.

That will be impossible to test with booleans, but you should be able to do it with strings, byte slices, and integers, as long as each of those is long enough to be unique.